src/TraitsV2/Trait.class.st

```diff
@@ -73,11 +73,6 @@ Trait class >> named: aSymbol package: aString [
 		env: self environment
 ]
 
-{ #category : #'old pharo 8 syntax' }
-Trait class >> named: aSymbol trait: aCompositionOrArray [
-	^ self named: aSymbol trait: aCompositionOrArray package: 'Unclassified'
-]
```

Comment:
Maybe this one should be deprecated instead of removed

Reply:
the idea is that methods that un-conditionally send non existing selectors are dead code. By definintion, this can not have users